core/src/main/java/jenkins/model/ModelObjectWithContextMenu.java

```diff
@@ -268,6 +280,13 @@ public void run(JellyContext context, XMLOutput output) throws JellyTagException
         @SuppressFBWarnings(value = "URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD", justification = "read by Stapler")
         public String icon;
 
+        /**
+         * Optional icon XML, if set it's used instead of @icon for the menu item
+         */
+        @Exported
+        @SuppressFBWarnings(value = "URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD", justification = "read by Stapler")
+        public String iconXml;
```

Comment:
New code should be compliant, just mark as private and add a getter
```suggestion
        private String iconXml;
```